doc/cascadia/SettingsSchema.md

```diff
@@ -45,6 +45,7 @@ Properties listed below are specific to each unique profile.
 | `foreground` | Optional | String | | Sets the foreground color of the profile. Overrides `foreground` set in color scheme if `colorscheme` is set. Uses hex color format: `"#rrggbb"`. |
 | `icon` | Optional | String | | Image file location of the icon used in the profile. Displays within the tab and the dropdown menu. See [Background Images and Icons](./SettingsSchema.md#background-images-and-icons) below for help on specifying your own icons |
 | `scrollbarState` | Optional | String | | Defines the visibility of the scrollbar. Possible values: `"visible"`, `"hidden"` |
+| `suppressApplicationTitle` | Optional | Boolean | | When set to `true`, `tabTitle` overrides the default title of the tab. When set to `false`, `tabTitle` behaves as normal. |

```

Comment:
Can you explain a bit further that it's suppressing any title change messages from the client application? It's took me substantially longer to realize this than it probably should have.